asset_dashboard/static/js/components/tables/ExistingAssetsTable.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Cookies from 'js-cookie'
+import ReactTable from '../BaseTable'
+import existingAssetsColumns from '../table_utils/existingAssetsColumns'
+
+function ExistingAssetsTable({ rows }) {
+  function handleDelete(assetId) {
+    fetch(`/local-assets/${assetId}`, {
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        'X-CSRFTOKEN': Cookies.get('csrftoken')
+      },
+      method: 'DELETE',
+      mode: 'same-origin'
+    }).then((response) => {
+      console.log('response', response)
+      if (response.status == 204) {
+        // TODO: this reloads the page but clears the user search...
+        // Need to come up with way to reload by rehydrating the previous state
```

Comment:
Is this true anymore? Seems to be working fine for me, and I would expect the session storage approach to retain search stuff on reload!

Reply:
no, not true anymore. i've deleted this in the code that i've finished and about to open a PR for issue #114